keystrokes/src/components/Character/CharacterDetails/ManualKeybindModal.js

```diff
@@ -0,0 +1,251 @@
+				...keybinding,
+				Key: '',
+			});
+		}
+	}
+
+	return (
+		<Modal open={isOpen} onClose={() => {}} className={classes.modal}>
+			<div className={classes.manualModalBackground}>
+				{loading ? (
+					<div>Loading</div>
+				) : (
+					<React.Fragment>
+						<Grid container justify="space-between">
+							<Grid item>
+								<Button
+									color="secondary"
+									variant="contained"
+									onClick={() => {
+										setAllKeybinds([]);
+										setIsOpen(false);
+									}}
+									size="large"
+								>
+									Cancel
+								</Button>
+							</Grid>
+							<Grid item>
+								<Button
+									color="primary"
+									variant="contained"
+									size="large"
+									onClick={() => {
+										setAllKeybindings(
+											update(allKeybindings, {
+												[keyBindingKey]: {
+													$push: allKeybinds,
+												},
+											})
+										);
+										setAllKeybinds([]);
+										setIsOpen(false);
+									}}
+								>
+									Finish
+								</Button>
+							</Grid>
+						</Grid>
+						<Grid
+							container
+							className={classes.paddingTop}
+							justify="center"
+							alignItems="center"
+						>
+							<Grid item className={classes.keybindingOptions}>
+								<TextField
+									className={classes.button}
+									select
+									variant="outlined"
+									value={keybinding.Spell || ''}
+									label={'Spell'}
+									onChange={event => {
+										setKeybinding({
+											...keybinding,
+											Spell: event.target.value,
+										});
+									}}
+								>
+									{Object.entries(Spells)
+										.filter((spell, index) => {
+											return spell[1].spec.includes(spec);
+										})
+										.map((spell, index) => (
+											<MenuItem
+												key={spell[0]}
+												value={spell[0]}
+											>
+												{spell[0]}
+											</MenuItem>
+										))}
+								</TextField>
+							</Grid>
+							<Grid item className={classes.keybindingOptions}>
+								<TextField
+									className={classes.button}
+									select={!!keybinding.Spell}
+									disabled={!keybinding.Spell}
+									variant="outlined"
+									value={keybinding.Target || ''}
+									label={'Target'}
+									onChange={event => {
+										setKeybinding({
+											...keybinding,
+											Target: event.target.value,
+										});
+									}}
+								>
+									{keybinding.Spell &&
+										targetting[
+											Spells[keybinding.Spell].targetType
+										].map(option => (
+											<MenuItem
+												key={option}
+												value={option}
+											>
+												{option}
+											</MenuItem>
+										))}
+								</TextField>
+							</Grid>
+							<Grid item className={classes.keybindingOptions}>
+								<TextField
+									className={classes.button}
+									select
+									disabled={!keybinding.Spell}
+									variant="outlined"
+									value={keybinding.Mod || ''}
+									label={'Mod'}
```

Comment:
seen this in a few spots just an fyi 
label='Mod' is sufficient the {} aren't necessary for strings